### PR TITLE
Refactor top Metrics/AbcSize offenders

### PR DESCRIPTION
**Pull request by lumeet**
I thought it's time to take some steps to make `.rubocop_todo.yml` look nicer.

### CODE REVIEW

File: .rubocop_todo.yml [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
 # `rubocop --auto-gen-config`
-# on 2015-12-18 21:31:20 +0200 using RuboCop version 0.35.1.
+# on 2016-02-16 19:29:34 +0200 using RuboCop version 0.37.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 214
+# Offense count: 216
```
Comment (rrosenblum) [inline .rubocop_todo.yml]: The offense count went up? I thought that the commits were reducing complexity?

File: lib/rubocop/cop/style/extra_spacing.rb [hunk 2]
```diff
@@ -64,6 +53,19 @@ def autocorrect(range)
 
         private
 
+        def check_tokens(ast, t1, t2)
+          return if t2.type == :tNL
```
Comment (rrosenblum) [inline lib/rubocop/cop/style/extra_spacing.rb]: Is there a `tNL_type?` method?

Comment (lumeet) [inline lib/rubocop/cop/style/extra_spacing.rb]: No, that's a `RuboCop::Token` object. To be honest, I don't know much of that part of the code and the refactoring is a pretty mechanical method extraction.

**Comment by lumeet** [comment]
I've rebased this, just in case.

**Comment by jonas054** [comment]
:+1: All changes look good to me.

**Comment by bbatsov** [comment]
:+1:

**Pull request merged**